edb/server/args.py

```diff
@@ -85,8 +85,42 @@ class ServerTlsCertMode(enum.StrEnum):
 
 class ServerAuthMethod(enum.StrEnum):
 
+    Auto = "auto"
     Trust = "Trust"
     Scram = "SCRAM"
+    JWT = "JWT"
+
+
+class ServerConnTransport(enum.StrEnum):
+
+    HTTP = "HTTP"
+    TCP = "TCP"
+
+
+class ServerAuthMethods:
+
```

Comment:
This is needed just to make typing possible?

Reply:
Yes, and also to simplify rendering back to the command-line format.